Copy to SD Card root directory to update/Language Packs/language_hu.ini

```diff
@@ -1,312 +1,312 @@
+label_read_onboardsd_error:Alaplap-SD olvasási hiba!
+label_filament_runout:Szál kifutás!
+label_disable_steppers:Mot.tilt
+label_xy_unlock:X-Y Felold
+label_start_print:%s\nfájl kiválasztva.\nElindítod a nyomtatását?
+label_stop_print:Megállítod a nyomtatást?
+label_is_pause:A művelet tiltva van\na nyomtatás alatt.\nSzünetelteted a nyomtatást?
+label_m0_pause:Szünetelés M0 parancsal
+label_test:Teszt
+label_deploy:Telepítés
+label_stow:Elhelyezés
+label_repeat:Ismétlés
+label_nozzle:Fej
+label_bed:Ágy
+label_chamber:Kamra
+label_fan:Ventilátor
+label_bltouch:BLTouch
+label_touchmi:TouchMi
+label_1_degree:1℃
+label_5_degree:5℃
+label_10_degree:10℃
+label_001_mm:0.01mm
+label_01_mm:0.1mm
+label_1_mm:1mm
+label_5_mm:5mm
+label_10_mm:10mm
+label_100_mm:100mm
+label_200_mm:200mm
+label_1_percent:1%
+label_5_percent:5%
+label_10_percent:10%
+label_percent_value:%d%%
+label_5_seconds:5 Mp.
+label_10_seconds:10 Mp.
+label_30_seconds:30 Mp.
+label_60_seconds:1 Perc.
+label_120_seconds:2 Perc.
+label_300_seconds:5 Perc.
+label_white:Fehér
+label_black:Fekete
+label_blue:Kék
+label_red:Piros
+label_green:Zöld
+label_cyan:Cián
+label_yellow:Sárga
+label_brown:Barna
+label_gray:Szürke
+label_orange:Narancs
+label_indigo:Indigó
+label_violet:Viola
+label_magenta:Bíborvörös
+label_purple:Lila
+label_lime:Mész
+label_darkblue:Sötétkék
+label_darkgreen:Sötétzöld
+label_darkgray:Sötétszürke
+label_heat:Fűtés
+label_move:Mozgás
+label_home:Végállás
+label_print:Nyomtatás
+label_extrude:Adagolás
+label_settings:Opciók
+label_screen_settings:Kijelző
+label_machine_settings:Gép
+label_marlin_mode_settings:Marlin mód
+label_feature_settings:Funkciók
+label_sound:Hangok
+label_rgb_settings:LED Szín
+label_rgb_off:LED Ki
+label_terminal:Terminál
+label_leveling:Szintezés
+label_point_1:1. pont
+label_point_2:2. pont
+label_point_3:3. pont
+label_point_4:4. pont
+label_point_5:5. pont
+label_bed_leveling:Ágyszint
+label_bl_complete:Ágy szintezése sikeres.
+label_bl_smart_fill:A hiányzó szonda pontok\nkitöltésre kerültek.\nMegjegyzés: Mentsd el!
+label_bl_enable:BL: be
+label_bl_disable:BL: ki
+label_abl:ABL
+label_bbl:BBL
+label_ubl:UBL
+label_mbl:MBL
+label_mbl_settings:Kézi rács szintezés
+label_abl_settings:Automata szintezés
+label_abl_settings_bbl:Bilineáris szintezés
+label_abl_settings_ubl:Egységes szintezés
+label_abl_settings_ubl_save:Mentés szlotba
+label_abl_settings_ubl_load:Betöltés szlotból
+label_abl_slot0:Szlot 0
+label_abl_slot1:Szlot 1
+label_abl_slot2:Szlot 2
+label_abl_slot3:Szlot 3
+label_abl_slot_eeprom:Emlékezzen a következő\nújraindításnál? (EEPROM Mentése)
+label_abl_z:Z Áttűnés
+label_p_offset:P Eltolás
+label_h_offset:H Eltolás
+label_distance:Távolság
+label_load_unload:Betöltés/Kiadás
+label_load_unload_short:Be/Ki
+label_touchscreen_adjust:Igazítás
+label_more:Több
+label_screen_info:Infó
+label_bg_color:Háttérszín
+label_font_color:Betűszín
+label_percentage:Százalék
+label_percentage_speed:Sebesség
+label_percentage_flow:Folyás

```

Comment:
Again, here flow means a different thing. You are mixing again verbs with nouns.
"The river cannot FLOW upwards." - here the word "flow" is a verb, that is the Hungarian translation you used
"The FLOW of the river is 1500m³/s." -  here the word "flow" is a noun, it means quantity, that's the meaning of the correction I made before and you changed it again to a nonsense. The label refers to a quantity percentage.